From d0d1b82fc90fc89b8fc4dbde2db5ba20d21ebb50 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 29 Oct 2026 09:06:52 +0100
Subject: [PATCH 1/2] Update the upstream signing key

Refresh newsboat.pgp from keyservers/WKD: same certificate(s), with
self-signatures our copy was missing.
The commands are in the comment above its Source line.
newsboat.pgp was in the lookaside cache, which a pull request cannot
update, so it moves into git (dropped from sources).

The current key is needed to verify the release signature with
openpgpverify (sqv), which checks that the key was valid when the
release was signed.

Assisted-by: Claude Opus 5.5
---
 .gitignore    |   1 -
 newsboat.pgp  | 237 ++++++++++++++++++++++++++++++++++++++++++++++++++
 newsboat.spec |   4 +
 sources       |   1 -
 4 files changed, 241 insertions(+), 2 deletions(-)
 create mode 100644 newsboat.pgp

diff --git a/.gitignore b/.gitignore
index 27179e2..1c377bc 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,2 @@
 /newsboat-*.tar.xz
 /newsboat-*.tar.xz.asc
-/newsboat.pgp
diff --git a/newsboat.pgp b/newsboat.pgp
new file mode 100644
index 0000000..8c8bfbc
--- /dev/null
+++ b/newsboat.pgp
@@ -0,0 +1,237 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+xsFNBFm+gDsBEADUX5D4Awx2XjHZJay5Sfd1+Qp685iGFdMWEZKjp3DMJGL/crUK
+2Do72DzXbp62lE3AH40IhivJkqSkq1UiekbIK6mjK6aAHLQzaMTP8RzbVxvYGl/h
+IAnBE0CNzqVLSx7g5dwTVQ1R2cO2HtXM9y2qqkz1lU13g17zNYiVHqukaJLjuj+4
+QGpuOzapKu+4UNyZ0qtq+nLoFE44ENSn4FFgnv3kka2MTG5vSYau7benGQ3EylcM
+usJR8MDV1QLHjAt+yLL9G7tQunOCGB9D9RRHfSqzsRyVMP7sZI0j/lguXB0TTjgt
+Tt4Jn35+vJ+uWYhmE6h8qk7jMKkSBQmnXDMsBxp4NF9X0Edsq59R42sPpw3OWbkR
+6j5O6w2q+pbzLKbN4P7tcY91PNj/TbmzolOk921hmcPZKisxqO6Q+XC/gMRrOnvb
+KcjpiWZ7MidsAZ4lD0HPPDnILYz4uLW3aIT3olfO6IZQvqMzgwK/6ivd/tSDFWJK
+uC1M4edIUoKNPl2tG2EWiVIQjTCoOwd1nSvLVuA4S34Xshtc/HMg7mB8cggeJlZ5
+0CIoQuACqaOXM72t+p+Jhjyjd3k1lxG8o9Yqr3gHZtJcPK4KhGS4SYPWWhkI4U4s
+uUjgptWs5hcET4UTJPL0fkwm3dP5AxfT5ouw4vtVaKl9/WFNhGvmjFEbtwARAQAB
+zSxOZXdzYm9hdCBwcm9qZWN0IDxuZXdzYm9hdEBnb29nbGVncm91cHMuY29tPsLB
+lAQTAQgAPgIbAwULCQgHAwUVCgkICwUWAgMBAAIeAQIXgBYhBLixdWoN2/B2DOZ8
+z07WzWGTK56+BQJqwkEwBQkS9hf1AAoJEE7WzWGTK56+f2MP/ia0sCFruKaVEAES
+J5mpnoRuj1h9PGb/tJFFN2757uENas6HRcMxHC+HEL9dU0oEnVc3NJg6fAVEv7C+
+nJ7c1QcNnb3ad1v/RaFU2YcqOn3ewzjxme5oMp0LayAx2JmoI440ia6TbRh2T5aD
+cgHzJmJ6ug3Yp9iGz72N34Kk75rRpS1lrU1p+28H7AnF3gv75d7S8FcC8TAou9X2
+49ufui7t/7enoaP1m9IB9mzVGdO6weB+SENhguSibIsNPHa9fnwdS2De2O5rTGzm
+wymqmqkPZFfo7H8m9PxfIkeY8MYvK/NOudGXncZddu4RRt+HknpO62NenrcTrPon
+VySkSyL3fWVbrwgn1cpitB+FeojSvWTu0IKW5NJkZfTex64W4SAroR/rlXU3OQKl
+W/0cWltoxUgPoqLmWZO8CZ5dHakgWZYfq/NRyHMKlVTn/uYXnzyrXlBP4wPNuQvz
+MrBvJ4CPoD06zeyOvnKmDLxTjEsmffxKIofln+3JGaEis3CeJaX8cT93RMJpOwRR
+rVEAud1KSHLchZpFTbVGwQr+9J1umVr3A7X8K80Boz+4wGbY9zsZ2grTh1jEMYs5
+GSejbwyiz/PB2GSuKoJGmSVPz/zPOFBLncp9z2TNI43Xg1hkvYbngtRDPmjg+XVz
+AACwmeLWqLD9ER1CQTTl2WKbzdXbwsGUBBMBCAA+AhsDBQsJCAcDBRUKCQgLBRYC
+AwEAAh4BAheAFiEEuLF1ag3b8HYM5nzPTtbNYZMrnr4FAmjP/CsFCRED0vAACgkQ
+TtbNYZMrnr6UFQ//a+pvR82i3mvDkbCMpxizH+BaX3aaTOcYpuaxAJ3toiXWALNx
+OYWjj1OPSrZdjOIPZgsTvFhlKq4w9GjMK1ZhTiHZFB7MKhPh7FazXaKOmLpm72TP
+wiGW7tJWQ3gqhD7mQPFtsivRLWLOuhCuY9ODUv0UXvpN8XlE4cZxpwO+ZUwhyjWC
+i85yCmjmYTvfvjtav7m6J5PKNh1TI3jg1b0gWFLZXpk9VRcyb2KdnYgaz3uXCf+9
+2nnb/o9gdZHYULbUF+U/Q7mfJTs1JSm820XQNqAeYIoxXYlU5TcwmQOYwxUKhcgY
+8OdcQoH/IQR3guB6g6JaJM0jOM5xJU8YbJLI/jrA3aK9FAzVh9u850fgk/8WDHO1
+6jLwXC/qxVNL4/pLAhpoQSzsWUPS9Az5139R1KgN86+2Qd3XtNMBAUOo7ZtylBkE
+qoJaHl+VXSPqBjSzSnCHWR1fD8UeA+16AX1oDD4CoxI9wOvCYhSzgmEn3OpmRcYA
+2peExylSC/icaxC0dSciAYO33a8MP8gURUSunNS6U8sRQc3LnCFRloZYf2Wizla8
+qBFz2iGez3uOG2YBtSRiW7bmxgpzwNIZ7yT8xZftb2slIjqaVqh+DV9Hi76lYRqG
+q1/HSVkIsnGnEY66mlJz4XCIp708/E1lCV72CEOSxOg5z26Q67JF1icnFSbCwZQE
+EwEIAD4CGwMFCwkIBwMFFQoJCAsFFgIDAQACHgECF4AWIQS4sXVqDdvwdgzmfM9O
+1s1hkyuevgUCZRXcvgUJDTiQAwAKCRBO1s1hkyuevoOOD/9FspD54iKR7/QUMWD/
+gHNhe/gQkHfPEiOm6qmrlikK/G/613ZDu5vd1DPP1GDLLuI3Bi4vvXXXuWhFsw7G
+rrq4OL6adPxMjCVFC2jm8O/5jfMZCTVduRF5ji4jsbGVZ2/DsG5ZXlThBUnQ2dQZ
+G3Jp4EEFMQbyY5nrUlRDfUVZbM8fnGLAHy13fKACnrXKYqHkDFQ3inMvSpYVMY24
+JHCWs5puIdTH4aiyCinSCDBs1iQKofYPN3a83EWbRcyXJFbq6JJo6RLs1BGpq78Q
+fegvC8Q3F85fHXXtFWXfH37yu4wXdT4dYMltm3BCHE+gHKKGRX8tF8Q4SWWOdPaN
+KljUDg0y6wdyTREz7z6Lx22f3skJa3HGo14RJT49ur5sRnVi/+x6+xM42KhB5+/d
++EaT9JvGx8Z8B2P7Qe2P/1bSuL65HfQLDwjOCCzDhJEeoSpLqBOlj+vIn02jS/TB
+ItpPFvTM2LqMjlete8gyOglbRwkpJHsWCATMfIFxtW1vrECsTbnvlSF9SITE6Tv5
+NqjYHqUWQQhA9vwlmT/454t36WjPi2nGyEbUnxXWIDCn/Knr7BCItJMS5Y+iTCxS
+yF6NQsxmtGEv2sAdrmMYjNgIV27xQ2ks4MaLbyGpQHUuVUxcMYYFW2pFkU11ggxL
+Rs4uSP6ngk4kN91/wTtNvNdw4sLBlAQTAQgAPgIbAwULCQgHAwUVCgkICwUWAgMB
+AAIeAQIXgBYhBLixdWoN2/B2DOZ8z07WzWGTK56+BQJjMJMmBQkLU0ZrAAoJEE7W
+zWGTK56+bdsQAMXS/xHb3PZXC5YbCjsGdPQHPZ0wDhdKZeeZZaVwHFGvy2ZqfvTH
+T2dqHbkIS9EeR+jKuTMdN9TIw5MT9zhV+lXbRPffhRJADC/cZJh7a1Y2WAykDV7Q
+xT8fBnli5NnMmUoaMy8b7a+Nzwb1/GzHbDcMfzSm7GFb9Gc/mRUACtHan/Vp9WXX
+LjyCie9zBEJtE0zUKiUC+wAwCE63eskORk3a0zpb6TaU/sIiLrIFSR+rkbIT9lqm
++jDX3slbhBj4b4M/BgcMKFTGz+JohKNY/cxBFzalgvoK2wcjbMMf/xVOxscibkO0
+yKwgDppyVHb3RtK7Q2FrgKouuprfTWAzPFJCd8+5xHHaeim0C3usiriyukSQXgiO
+xVti1511Var2ji/wRaHwHUjLStwOzeoZnwrHEF+1XSTqmRPHeJCdm0v0ieUVoPyC
+7/5tdhDUbAgREW/+JC1wWapV8iorkG6G3Rjt4DRmrOFZcYJPohAlxzOxY7j9Rw4n
+QUE7fSpS3CeEmZLFRItVbMeULyIkyXizylFO3OpzeHwbckwG/eI3vu+AW3GR8D/8
++1i2Ba3Jef+CA3glwogUIdb1VnHFJmHdavbYvcfqHZPrvCx9Hbik9QbAGNPSmagx
+B2LGiclngoJO5DoPZF34r5M3lXI+PuMvLfHEN7RaL2NhY9zwuSIqwqaZwsGUBBMB
+CAA+AhsDBQsJCAcDBRUKCQgLBRYCAwEAAh4BAheAFiEEuLF1ag3b8HYM5nzPTtbN
+YZMrnr4FAmFIkhMFCQlrRVgACgkQTtbNYZMrnr7s/A/+LK/rSSOr5RFMauK07KgK
+BNrBqOIGNXuKPP8r9qhMPv/Dh8ccYz/iigChZNWy229TmPJBNTyiFvA7CqvUoVu0
+uleR6bLKdYqDPwAxoQmk4RetbouL+NSpU6oEBKOGA9OIpUw7+cXKBR9oDaigR1+h
+6qHHlOer390ktYRbFmd/g3CwM2ZUsFgjHeofB+nIuHS3du9LplwesvHTF4xtUonH
+yruIH+Q5PrWIYJiGalJARiPRtxpfqoWnBcl/RFxBlTY7ATU1jdWsi6xKh2vwSzvL
+uS+Pl2QXh3fM3HS3a0/7TPfvaDgBPYvvodbOklziSC2dZ89ECjOS+Q3FhXwR4oeW
+eggYs33t3TtKgDtVw9oDPMQn1M9GV7iDj8+FILLsiZy5R9IgrPr985fERwEBuAju
+ueF7VzlKD+rKtyzo/i85Y0wg+o3xPTIkK/L35faI+f/p+WVt8bqI6BZxzU+Imju5
+HwgMJ3lNK2ocG+ULH1IrpRwxZ59bPSrcSX4JrENtr3A3GjMcSDr2Q0kY36zu2CTh
+5+2xY1R0DegqDS1iQZIwDBi+yY1vBiNvcuaCjtn8p8K4nDYd/JiuP36FrC/Ugs7g
+HsxI7kMo8yGNJkGZ3Eb8L2b8X2uVWRIafIS/CQCNQL/empneevbQUSwR/Jf/AC86
+p045vAEw9jXg/xh3CAWatXzCwZQEEwEIAD4CGwMFCwkIBwMFFQoJCAsFFgIDAQAC
+HgECF4AWIQS4sXVqDdvwdgzmfM9O1s1hkyuevgUCX2esxQUJB4pgCgAKCRBO1s1h
+kyuevvCtD/9sjfHcUjsYc3lGKrCdxvrv0qROTehkWqlx7WbmcQvsv4orZjZCNogX
+U1V4V7ggM6UM9OxYyusR+si1RoMKCkEo8ACw6/oKbndvKygEk5qHlc+OooDmcaRT
+/sS9nC4+gkAXyLMyl0sQxKMgYog0hanJpN8Fgy1hkEYDjFkwDDS3J0FCkUoOK7v/
+oG5Dtrz0CThfNBNW5P62m14bn1SL4mRZ3xdOVHekCG6POT1kxzt5He89DtQlCjjp
+vG5Q+Nzo/1DB2K+jkAL8Ueiq6juy9YQnK8WgpXw9Mf2/OmNYTFzFCRtCLbrl/4je
+bHcbWezOXyyFJyj1fpWIBqwvqse9av3TQyr3E6x1CjReh7XIG7OOvYXALqXkN7nQ
+XRXWnOhgII3CsPj2lY0eEUtpqlP8n7frLpuJacmNupNxPQNY1oTs+h7/lkjEAK9D
+ww7R3QvTDumJyA2BI1g6oDUUvGJaHAsiv9xN03oq68RZ7ixzpZ5HoH60rHt9apza
+1lR+q6etaNGcgJdDDL5Tc+KlZzt2vFC725uCiU8x78KYvoyWuU4mshJcZEqgmsuO
+4sdvtPMUT/aZN0GGmA2lXN4q+X6OfSsPUK4MQchi+WVNkMXyzYWstC/E6H8i8kzb
+AiQdtOsvOn9gtB0vwV2NB4ehQEtq6xPSmEVvumPJu37nAEKDaIdS88LBlAQTAQgA
+PhYhBLixdWoN2/B2DOZ8z07WzWGTK56+BQJZvoA7AhsDBQkFo5qABQsJCAcDBRUK
+CQgLBRYCAwEAAh4BAheAAAoJEE7WzWGTK56+asQP/iYxxhj4IxhfFh/ViCI5tiST
+HsSrQCngRJFK7UemkMAX2VGdy9jATk0p/Rl6DT8evicl9z3B30OeUDVfjZ6NfQSJ
+HjsFhocbZiA7IGJL6XDaI5JtQOyZw/VUWwhveX4PjS8skIy29VmmL3thsw7cbUjN
+nh6kvyYxekCY87MN12qFa27k2yk/smCyx0nLAHay+dYKSDRCOx4XvoKnxExP92Yg
+y5NGbZvpWXg9sOmLtczONw/a9ytTlzU5bjC7bprNtSHNvIgDucuFDKXgIqx5IzeE
+/7kPG9l2yL/KcTuRYpFOiGeocrBXLgFEh8njuY4w5FXu0pOC+L/XQqgZbdlZMdeJ
+B6nZq6ltvlaWEoB05zwT2mezyuYpPKPOKBMhSdadPE+/ylslTPI6e1XbekkkU7fG
+SXcjFg4hZLcl2XAUxnfpDXC1wGcq2FdXjIVNCfMHKHsjjA7yrf5EJguyHcBJszXQ
+S6SlYARqackl3Ws9AdxtPEzd7A9e/PIsnkWS2uaHF1/Zft+eevzzclmP1MN/Io8g
+J4zo111e9IQYnqtgXtYN2HTYOqwKlNddxw+DvBTKrZ+3wvce0WUNSpYlYoz9MoAv
+TTYp4f1Mi60cJ9EThkz3orjZQNsfyrIZ6/oCZItFdD1MPb7ygwUQ7IB3dygAGL8c
+KgNHC+mpKZ++0ddBLX9swnUEEBYIAB0WIQSLcENEHveODQI8ILsXFe6qFNrsFAUC
+X8c1pAAKCRAXFe6qFNrsFKhmAP95POODf4P38ISfgZ7VB5rO5g4vU2PuoqxA+eD9
+5nLcKwD/Za+uOkqkBjQ2nCbHr+B+PlrRsUL6oHPuJ1lnb9il8AvCwXMEEAEIAB0W
+IQQa/8zKKfnvTxMbqohQ+renrqtjkgUCXq9z0gAKCRBQ+renrqtjkqvAD/92pXXn
+Fs+4lYwWJ2zAF+WJG/Qp0/MzLGPoXmUc5WDZIXW01S1atUeSM+hhJUgB/zMzFgIy
+8coPt4ghQ99GkQXCIti0Tw7CoJJRTQBeiYV2Fy1qISqt5r5LzWi2D80KC9DnVE3n
+XYgpjnFfXalcPyN8kE9ZjJ5d9gMqT4zA/6U3X3OjuGvI/aCzv+mFV8pQ6+jL1Yq6
+p3UKxusNYOrPbTkplp4WhGfFrEii2DkzQgWyTaQrvk0h12rKmlrmZvxTInTCxgzR
+uRSlwTiTmaYEX4ZaxTjceeWGX1UPvyoEndw+Mq5P8O9zUJ4+uKiciKLMLkwIcEUT
+HiGrXhqmbgsZp7O3OGJKQC+N194hWJksb8qvMk/P2WnRwTNUxgj51C5xP3kC8wAN
+ADHZ4w1qKpPySaatxGfaVgucsbx5vb/yyVDx501GLsE76Rxe7KHW0iuyy3hUWRYB
+W2FApsC408wVbRK74bEX8/hsj7fYuNfCtsAJtB/y4ar3930t6r6QlIIBWoA2hy1n
+TzODt2py/VPLsz6FfSix1QXjDKPdgIC8z+4QNdLGQZBPHpRndinHymJKzIShQ4DS
+MpbR16RXPYgG+1iCMgTFE+7x5z/c+nSwKUOAntH+iwhmazy3+5vfxifd3ZtnznJn
+aoLf0Xg+/4TUdl6lnLezyz8BSskx7mgPijdxa8LBcwQQAQgAHRYhBJPr9EYQd7pF
+7c4RGbdUrR14nIyDBQJanRdeAAoJELdUrR14nIyDgoAP/i8l+c0k9mIj+WDcr/Ov
+ulLq1NIB4v4bc8CLW9nODjHABUJc7s9rb39ShcadPCnaewwRIdGFS8kSWQRKDlH2
+vGSRVzJpJMKx2rEaPB3I5IYWsuPGYpsghGCn3jIC7Duxwb1wRcsb4ubO6j9Fppy4
+XL3Ns5w4PFsGvBTPxKVi/bDt58JLXsJgJ491tgAExSVw1QNzdbaPS47uZrAF3Otn
+E/ZMPYBsecz7o5/q2SzWKlFf8iC/LhxJBGe+oiZ6byvmr0rTJWd5ETlXx7yFXcii
+G8UxjyIeMCnn4uI9tPDxID+sUnEag+AEbbkH14MIZzPoJ5Ao3dGnqrgxJI68dqm/
+pUvcIgmC0EneJAD24YWpoSHVA83cW/7ZuADb6cg4lwzjlPQyTyPXS50lAjnzMc2z
+Ihc8ZTPoJvvjdRS/n/BTQMw7C56HikQXN0BkdZQ2qclfgnGGUbwgn83RODnjQbwv
+XAm+8o0EHTlxkb5JkVHpvY4k9oCTNdtDSYVxZnxwMI23ClCApqRRDaydj60PCWBh
+/goLxF00a5YlNG4ngNgf6SYoTdQBd4DIjC836YuR9/G6yInWL93dhWxAcWpA/dIK
+8cdt6MRctcPt7TSS0FnV3IvJIfEcBfkq7TBBGniepL4QaAyNnM0nV/Lg0BUBYv5g
+3WGVGT2nqdXoYTZAmoue6XkiwsFzBBMBCAAdFiEEzmxDB5NIWOP9lKAPNWlhogyL
+/QMFAlm+hzwACgkQNWlhogyL/QNIdhAAmSVR3NAvOcdVMXJrF6fElG0QG/lmuWdM
+mrHpZPNtH7N/9w5AIHhQ/V7ms+NrBD9+mnb/DAGNekXt7DTHMsYbtc/6sui40u8D
+AGXi9e0X58wStiGDFSptxOjAYzlawIQK3+QVlFayMbGQBq2czo8SU/waQ3FMJPd9
++QANb7e/96aIYhMrcizQXHnhCyclGFkuJnlgsiGFD2y2k/U1bCAa2a8WgKJUBrHt
+2KDK6URAk4rFolR97eB+eqXHsJQ0ws/RJxXKTlOZlVOWJQLDIWOzfyCu+nCqlqwD
+X9+aj9Afxldm0REyECbGaMMSzujTEhTBE07OG7K5VLeYb0r0asyvPBSk5dPKeZ1u
+BVExXTWzpL6sdPjTaAk1LhDLagia9lDSndP/nA6KaLgilC69XKXlE9KLdwgT162V
+fvGFzgPUQtDYRlmmrbsDNGEWiaedJboKYuRfJYHpitBKQGqtwzdXZH3Fcr+NTlAR
+soi1d5TeL/aLuhJr6gSmSFcrqTa8xd5LhD7Ji0fYmThwm251P13oXPdPQ6Wjxm9/
+0regM/vU9Ut9WIBv8NiA+RCKfB9/sZsORI0+144e2T6++84mqCqYctGs9JV2kn+n
+aVJvlLQJWKL5QPCUrS6n+kfHQufK2jhqVGu1fQHWowg7ySdZXKLj+rruG0EAYIdE
+ru2WZVMk7HXOwU0EWb6AOwEQAJ7gO45kUFdQmAdwhLbbrl5G9gzroNHKXojgn2c7
+RQTWSX2tfqMh5zoqhXy6XtJd0UcEYHUDEmXKdOAU3qfl4trjY/rvQNGRxiqQKki2
+pjE3Io+UHUHF2gugexK2HJ61sPWzEERL7NjqRw4S97coCXRgzHwdP03DCzO8dcx4
+wWdcEpYCjkfAZfbo3CNVVo3EQj0IPQc3p403LcfgyTu/WanWxvqczKA8nWO3yad4
+k0iRVbARqxzlaO6ImV0P6ChTeKq8lGZKzGGDbHUvtm3W9QZpWd9SGSBvlp0tUcP1
+m5tB3vHNYce97afKjVTCY18/PAjqrhB64G2a3b+ZILxL/ktKuHi6JQUfSebVA1ZO
+9Uy5HduR/81hRIRM0j0wF/i5WvmahkQNCeyTnmecd085zqaXfOJM0tndL83pKlxY
+p5riWugii4YDuFvCnn3l0eV39RKAPld88DC9SQj06bA+ddCcTwrSCIKWVhA2n5yy
+LVmamu35n+xDv6h9mSC/5rf5yPERmHmftQAghs5kvaogXtoxp0KdcZs12RKbL2Sh
+78uJxmsJ3HGtwP2stfchjUa29o4mYglCc7L/DjCiawTlBpymm7liISxHZTz/Rtz5
+WF52V/U2QpdYw08HnnnaHnbJHFsfGQrMoYU+l1JJQVkLiHc2EkUt5pMQCyXH44yi
+ysWtABEBAAHCwXwEGAEIACYCGwwWIQS4sXVqDdvwdgzmfM9O1s1hkyuevgUCasJB
+QgUJEvYYBwAKCRBO1s1hkyuevl/XD/4lN/eZeScmG+KQ41WS505TB4sSxAYcsv17
+HB3yawM6OkfRNMAx1ilFb+RRgoCP4AZsjLBlC8GsZRN+uXEMBAXoHOkaV8pVgwwn
+M8yGD+D5I7Jks9so8UZWscdKfxqZhW3XJIuAh6QSP/Z82ZJzLM+q8w9b3BQjHFDl
+ELkkfs9q6ju3tpC6dL3QULlOjzLXP15xjf181Tk8V4uKDQ9jG5kw05bLmLABLa1U
+ifxA4T+f7O2kjfcFOXOsbJuawP5Yc2avWflnane56NvZM4g7JGDt2y7de1kUBhW/
+U0QFL/zT9xpnDDIQ9BiqQRfOf1njHCdFCtAsT0aiKXKEdslhadWU4mh/gGSuL5k6
+E9l+YshmqbgAFV0yR+eFdLMj8UrMWb9Q+xPAedJ6wuCQ2zlgBB0wd85WF7gAE0ZA
+f5vYC1zlvRf3LOYnd/RXHKAt4Ve+yqGJjkzDoOxnI17G4xVhswKOCSyUeyfpXLxu
+P94lY8QHE0wW/BdA6uiN5qQJzrWHMzLngklg7jXNu4fY8p4xKaAbHgGH993fr7TS
+ZHv3aw30R6tMrVweBnPVY1a48dwegour1eu6fiNyJEt2CqlnaTCcw1B8Ej52vo9P
+LP7T3gYRQAx1YKeRTCalhRhyLvGNKHbIoIAs1I/y2ostF+YJqkgV75CNcVQFoV1C
+Fvel0wDTocLBfAQYAQgAJgIbDBYhBLixdWoN2/B2DOZ8z07WzWGTK56+BQJoz/w0
+BQkRA9L5AAoJEE7WzWGTK56+/RIQAJe7QFLTOeONIWVvSu7ZPczekHTe02eeOFnB
+IgOvEFHaIMotm9OJ0cmeVtlAt1QKjcHtQKXCrsNmYPrU5bLFrk5y0unCOQOgRWy2
+VAlP2XCYSNr48nhDtBkNWN8yRieEgtQM39aXNSfUm1OVGxzxPYq5UwLT7zCxOu48
+1bx8k9Lzr52KrGWn6Fr9cXgSb11d07R2+31zc5VrAB5nJQ9IfKsmCcW+LxDa6UDr
+n/ibR2ekkF853aKVjx8bf/Ym5rOilSBjtitlMg2H1VyEtkcEax1IiFFjZZdbwh1f
+hIFGl2F+NFo1kJv5V2T7cZ/hpTJklVDywoeVHANx+QOxnPZWM3NZBVO3pdN2wnEX
+IEVl3lnzpR7B5nsm+aXMwA7OUKeTmHO4fCd2LjLwCdT5Xxp3UuqU3f+T6WnQMra8
+okWz0OQLS89MtvlUJgyLxmze5VJDvIzs3uTGAwtt+Mq/fTf2SdizTG58PvqF/esn
+6WLIfqihPbMSa0NKe9IXYd1CFkrUpIzI8TVjMFbBkeHQzitjQ2CPq7e37PW1KmVf
+oc0+VXouFTEAadV4UTJEG9FOU9RY8kSKrMSEOVu/tBxI4TMOUjeDLGByJR18ygax
+AM4/qVHh5ePZq64W1VE431bTVB4JF9lsDTsiZ/Tqr8qmxpMfiX9OYpb1vbq3c9/+
+UKwu9hu7wsF8BBgBCAAmAhsMFiEEuLF1ag3b8HYM5nzPTtbNYZMrnr4FAmUV3QYF
+CQ04kEsACgkQTtbNYZMrnr6Vvg/+OvkhHEl4LgdH2NyVAhdTzV5fsHLVWMUGGcLR
+5VAUCkRDF9K3PUF3yN3HRC48tCq103Plyv4dwDx4eO1ZIpojnYFmSO2IpPxPvAgJ
+uC7t2A9Lwo58nN/5/7h1dbzUXfVmPEZ8oMJXNF+X/b5rktKVeTqZTVCrhOq4zpJP
+XgV5V0tdd355ipuCQExOhR+WmBuZ356Aa0hoBVepbL/beh5moRozqyOaWNHoExGn
+sYh6QaJHHwqdix8GYEswVqFsHbQnhwspzpcNmOGeA+jea1CriiuK2iyZX/vTZ3IJ
+xQiYR1jojTj7BkSzgxf9wJhwIBKOO433UKqbp61Tl2bNiMpkLqmgjA7M9tgxLbuM
+do6seAWOJe6tfgG9sLNXUR8l2bXCkcjuCAeJS6noxBWZdI2LEZa6w2ZfbMWFtWLb
+q6nXQ3pG0RCAMBUkquFG6Gsu6TadOnYN3fPF1h+wi3uLt1hRlMWtIqYyACh4Pqmx
+U9uUXIEl4T99KtZeO6sz0EcogmKbSg1KuqOSpJb+vyZyBeURgGAFKY+SAdF1wUOh
+n81qyGntgIx3GiHuRsAuKxDCWF+JUf1ffHVQBuhZ0rDZKnCZp8JECUyC8rHggoDi
+j3HfrIE6BhrBdp3zS6NbcMtaAbNAEA2ToPFlbyWLL9MuatRPdP/wFcvwEUOfe3VZ
+8+thLAfCwXwEGAEIACYCGwwWIQS4sXVqDdvwdgzmfM9O1s1hkyuevgUCYzCTVwUJ
+C1NGnAAKCRBO1s1hkyuevhz4D/sEdf2apY1ztfMJvzFcDwUIe7ahIbQF0ROO6435
+seBc34kKyMyQRQQ4N5v5uN+NM6v8Z7g1f7mbleBOiRp8/wwP+2PPa8QmAuYoheMg
+2B61jp4Va9ZI5wjUQuxj5mMcQnmoxbYJAAV29+4/A/DlGR6853Wj6eW3MzEKcy5U
+jHwxL1puaY7obDUMMjX4izD67nWOasOYQcaXm4TNIMa6bCaRuzW9Pcs4SyEWn19/
+6QmzZ4An9NYFdUGLA6w+DdyiXD8bTSsjDSByOHPmg1U0wfCzAhl1gcdgdDc2gbzP
+CfuLJ1I3R70rAaIvbPbTv/wYIk+E1mH7JLwz12zrQZ0WkNbzQM6in7s8lQcYO/YO
+5rM4rkpSMAQhX9I8sqcY1+6+QZt46UDljgOx78JHFl7ZtZUF+UUDcsck716uT89t
+asbCLr6aqdLCJC39/0pHkRP6PalxPzlYj/PTuTf0mD7rpCNrwNG+uDllDHcHKUf1
+dKJHSkUMDQMTxlv1Ok5VZSBNjWIOuqL2bzK0sVNysuhpK3OPIQLNgQi3vG4kldBt
+ncz7uJgph1uKGoMO42a81QZIjQb3wHAPxQPfs/MLqqj1JyQrEnoriFWAEAA+M34h
+FwZBaFP6ZeJQW1z2AFt/2jqu1Ito2UGqJjBAxgJ1pH2hcZofhGQxfINDn4S2UY8B
+wJ+dW8LBfAQYAQgAJgIbDBYhBLixdWoN2/B2DOZ8z07WzWGTK56+BQJhSJImBQkJ
+a0VrAAoJEE7WzWGTK56+EA0P/A4q3smn9KIWeQNmMaX3IByEaxv87WvLRP7Ze6ea
+T7XpUVBdCoVtPCoQnQhwM5TRbt41udtGHlqrz9e2WWJCcg/HRM0HeHFK3JdD78ZP
+y+Uk/9nfy6kV/vX6d07XotNRycIT6Ygvxpis6OCi77uF3HdQudxz+5yWEc/q0NUz
+IoaHLm3lXsqHeb2KKcKr+Az7NlRQrdH0lsb9WeYY1iQxld6FoACLSeLBFMDdFepL
+TNMS3fTsZ7AFXTjxlFfWv5xM0lPVYnrn72BF9TFAGYZmbgO0iyNOv/ofaW6wNdkR
+YQIxI4WZukdj67DO2MdNjKwKIQmSFAKPNT0qxboxiptXvjCwVSbGVTwaG4ejmBEy
+IwCzU+r9N2YT1N89D4WQyY36um1UkQttKN7OVj80cwUKVO5UToY+iy7fZxGJfDhD
+pDIz4YoW4KxZ2MKh8As9fc4ILG70Z1W9byx3ROxU5m9PJySAx7pYY9uRp79fwlMo
+fkLsOBDD+VJyXKQy3ougl0QisoGbduxuyZqvMDExyBPdA3aBM99j4DscG3ing5vB
+B2i8SyDXrnu/BJ85fhEbWyZxWrgqTTAYwGsuo15Jr0Smgg8aahN245OXHZsiDF9O
+th6WVFHF1Fsd/zFpJCnlFRkoTsIcxofnokD2T+hWDyp4Y0jULsHjZNiI/wnVugUz
+yyskwsF8BBgBCAAmAhsMFiEEuLF1ag3b8HYM5nzPTtbNYZMrnr4FAl9nrLcFCQeK
+X/wACgkQTtbNYZMrnr63bRAAjhtxgZqGhzKLTUAUHj/4+ElzYnhY7rn0tKfI1Hdm
+0rRFpI/Chy+2Shcskvbh3fEvEUQakCfXg+VBTLmX6riTuGJZVy4MIrB37V0Wc61H
+TOFDedgvJvj6xsdXV1xAe7XvgUVtlVq3munQiKryUOl6egc3noH4D+215j9bEaXU
+exQEE08gTRjV8gIMIMYmixFrb1fK51HNxoXpx5ev8a4Y4aJK3nBuNnT7gpOeqRY/
+lfs8QLLqQxLy0Hq8QztCihWbzGbAgdvmLtqzpazcUpxwQutBYIplXY2L3ScVKv1n
+RautGRt0wA3qOHfB3ReFWTuOLa2ip50+QC0au1YmZMyaZvFvlMTzI3WZ7xynAuzo
+JN3usnTaDSM6Hitwce+Prrb6NBiqELR9B9NDCUGKSHNmIgTYxpkpJn5slONUG4XJ
+FQKpzWbjXwjKorjiXLCARrXaRduaR0gUai+zJTl9d92jPJMpTY66YnEGkcgt+QrJ
+PU1UmxFfqXmB2uLwIu+jUFVvJ2CsruJk3dhvLHVvJrRGOHXQ4WUzUhg1n97JI4rF
+zgpYBLG56qGC5ka9tCsVBDzc2IJRUOAMmr63XibbeWmxgDPudovODqxtP7z/WuG/
+NTVH9+Plo+3IofFI6//8+uFQHxx/nE+aUQ88qf/KbpgqcjcksH2beDs1dRNp0H4r
+DEbCwXwEGAEIACYWIQS4sXVqDdvwdgzmfM9O1s1hkyuevgUCWb6AOwIbDAUJBaOa
+gAAKCRBO1s1hkyuevgQcD/0T5zWVsGHdceB5PaNOVoC+lx0wo94a041qD6Ysac35
+j0gwJP5qCaB+MKBkJuucksO14mexKt25bl5NIq8yDvHdB9zdFn0RWX7WhJ3cgPfD
+yzSGDRRbaFvVyr9c2qWUBsKOT+WEBCL5mwDMiQHZeiNgsLC9WT13x3fdG0KmukTu
+M1mE7YFLclpRbDCo1FILogUpLRQ/+JbuDudXraSIvkZGMbE5UQf1jiFQvTsKam9c
+mCDWQ46nRGT9U6mSbwOxjY7agUyS8794o/3lyDIk6nOfLnYtk2mW2UmYcc1PH+42
+OeyLY5JJE73OV3+deQmLvEbknqZkmZdl6zWcwDIuB4B51WcBPbdh2wqsF8DjcZa7
+RpKM+LRiEUlRZMY3ZzAaGBBeQ4yJKsfQz9n0ZhRZhhEueodOPFJOCRl6hbXXbr7t
+cHvnNP/P7LHK9MKb3NXDkCinnjLC+oZhKRnzfmqXbz1BVCHPaBSQ3aK+NQHVHBAC
+ByKBCL+qRoiXmycq1/pKdUHsJLolAKeZrvn3d4ZIUB5Ly1ujyEGyFTdMtRmf5iLt
+OsxFAJSX4LYUjPjy1j4o86bUhUHyml9RaX6FF7tDHbVjzVl1KkN/4vsUk0O9G/VO
+B6sh/chZxd+URLGIGzICbTps7C6PmMHZGG89uMn8uU2huMEBVUct0PNUeuzabzlt
+QQ==
+=9t0J
+-----END PGP PUBLIC KEY BLOCK-----
diff --git a/newsboat.spec b/newsboat.spec
index c56ae04..2c1d143 100644
--- a/newsboat.spec
+++ b/newsboat.spec
@@ -10,6 +10,10 @@ License: MIT
 URL:     https://www.newsboat.org
 Source0: https://newsboat.org/releases/%{version}/%{name}-%{version}.tar.xz
 Source1: https://newsboat.org/releases/%{version}/%{name}-%{version}.tar.xz.asc
+# Refreshed from keyservers/WKD (same keys, newer signatures):
+#   sq --home=none network search --output - B8B1756A0DDBF0760CE67CCF4ED6CD61932B9EBE | sq keyring filter --experimental --cert B8B1756A0DDBF0760CE67CCF4ED6CD61932B9EBE --output update.pgp
+#   sq keyring merge --output newsboat.pgp.new newsboat.pgp update.pgp
+#   mv newsboat.pgp.new newsboat.pgp
 Source2: https://newsboat.org/newsboat.pgp
 
 Patch:  0001-make-do-not-require-Cargo.lock.patch
diff --git a/sources b/sources
index 67d52d5..8eee17c 100644
--- a/sources
+++ b/sources
@@ -1,3 +1,2 @@
 SHA512 (newsboat-2.45.tar.xz) = b72c5ea197d138718c2854010c61bc39ff7376a8fb624cbb17d1c9d91c03fc1e032c4640374da0b758d2994a2a31ff4c965c080eacfcca89fcee34f3cc7012e6
 SHA512 (newsboat-2.45.tar.xz.asc) = 923a797b328dacb1b5b1261140a703296aa6846911402b6e3a17e81a287f3952c6ff5243833f4613a8071bde9e27242b0869f2c73887a3a26034829b16796f96
-SHA512 (newsboat.pgp) = 4f4b9cf3768e4fc05a72670363c8b26159ad196e7a742e4bbfe08ed4c0e751ea4a206ca067db12e5614800febe7f06a1c3418fa50ff05aa5f086efc28a77475a
-- 
2.56.0


